### PR TITLE
Upstream merge v1.0.4-stage

### PR DESCRIPTION
**Pull request by willrlin**
## Issue

Addresses #145 
- [x] This change addresses the issue in full

## Description

Merging in upstream v1.0.4-stage. Conflicts were in these files:

CHANGELOG.md
backend/core/scripts/listings-importer.ts
backend/core/src/listings/dto/listing.dto.ts
backend/core/src/listings/listings.controller.ts
backend/core/src/listings/listings.service.ts
backend/core/types/src/backend-swagger.ts
sites/partners/page_content/locale_overrides/general.json
sites/partners/pages/index.tsx
sites/public/layouts/application.tsx
sites/public/page_content/locale_overrides/general.json
sites/public/pages/index.tsx
sites/public/pages/listings.tsx
sites/public/styles/overrides.scss

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/address technical debt
- [x] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

Ran all unit tests. e2e tests had some existing failures that need to be fixed.

- [ ] Desktop View
- [ ] Mobile View
- [ ] Test A
- [ ] Test B

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes

### CODE REVIEW

**Comment by willrlin** [comment]
A possibly easier way to review this change is to compare this branch with bloom-housing/bloom's `master` branch: https://github.com/bloom-housing/bloom/compare/master...CityOfDetroit:upstream-merge-try2

File: sites/partners/page_content/locale_overrides/general.json [hunk 1]
```diff
@@ -8,5 +8,11 @@
     "header": "Detroit Housing Portal is a project of the",
     "headerUrl": "https://detroitmi.gov/departments/housing-and-revitalization-department",
     "headerLink": "City of Detroit - Housing and Revitalization Department"
+  },
+
+  "application": {
+    "details": {
+      "countyName": "Alameda County"
```
Comment (anders-schneider) [inline sites/partners/page_content/locale_overrides/general.json:15]: nit: I'm not sure where this "countyName" is used, but maybe we can omit this whole "application" section?

**Review by anders-schneider — APPROVED** [review]
I looked through the files where you mentioned there were merge conflicts (and checked other files where I had context) and LGTM. Thanks for doing this Will!!

Note: You may already be coordinating with @abbiefarr but I think she's trying to merge her #142 today, and there will be some sorting out necessary, particularly with `sites/public/pages/listings.tsx`.

File: sites/partners/page_content/locale_overrides/general.json (hunk 1 above)
Comment (willrlin) [reply sites/partners/page_content/locale_overrides/general.json]: Done

**Pull request merged**